metpy/interpolate/points.py

```diff
@@ -170,11 +170,11 @@ def natural_neighbor_to_points(points, values, xi):
 
     Parameters
     ----------
-    points: array_like, shape (n, 2)
+    points: array_like, (N, 2)
```

Comment:
Is this actually an ok way to specify shape? Any better ways out there?

Reply:
To be honest, I don't know if a typical user understands this, as I'm not sure I understand why we have M, N, D, and P floating around (I'm guessing this might come from GEMPAK?). We could also move the shape description to the following line, so that nitpicky doesn't try to resolve it as a reference.